### PR TITLE
Add background to main menu

### PR DESCRIPTION
**Pull request by ChrisKrane**

### CODE REVIEW

**Comment by toberge** [comment]
image
The gallery doesn't work 🤔

File: Assets/Scripts/UI/MainMenu/MainMenuTumbleweed.cs
Comment (Fueredoriku) [inline Assets/Scripts/UI/MainMenu/MainMenuTumbleweed.cs:28]: I have suspicion that running this function every update is pretty costly (depending on if Unity samples their terrain from a heigthmap of some sort or by actually calculating geometry). 
Might not be a problem as there isn't much other perfomance heavy stuff at all in the main menu, but we might have to monitor this a bit

**Review by Fueredoriku — COMMENTED** [review]
Great work, looks nice 😁👍

I think we should take a look at fixing the gallery before merging (maybe just do it as easy resetting the camera position to its old position when switching to gallery (and then back to you new position when changing back from the gallery) if thats what's causing this? Fixing the underlying problem for the gallery is not part of the scope of this PR I would say, so I would approve a quickfix like that)

File: Assets/Scripts/UI/MainMenu/MainMenuTumbleweed.cs
Comment (toberge) [reply Assets/Scripts/UI/MainMenu/MainMenuTumbleweed.cs:28]: Since the terrain is created by painting on a heightmap, this should devolve down to sampling that texture and thus perform well.

File: Assets/Scripts/UI/MainMenu/MainMenuMoveCamera.cs
Comment (ChrisKrane) [reply Assets/Scripts/UI/MainMenu/MainMenuMoveCamera.cs]: Fixed

**Comment by ChrisKrane** [comment]
> Great work, looks nice 😁👍
> 
> I think we should take a look at fixing the gallery before merging (maybe just do it as easy resetting the camera position to its old position when switching to gallery (and then back to you new position when changing back from the gallery) if thats what's causing this? Fixing the underlying problem for the gallery is not part of the scope of this PR I would say, so I would approve a quickfix like that)

Fixed with a small tweak to GalleryMenu.cs :)

**Review by Fueredoriku — APPROVED** [review]
Very nice, great first PR 🤠👍

**Pull request merged**
